dspace-api/src/main/java/org/dspace/submit/step/UploadStep.java

```diff
@@ -614,7 +614,7 @@ public int processUploadFile(Context context, HttpServletRequest request,
                 if (configurationService.getBooleanProperty("submission-curation.virus-scan"))
                 {
                     Curator curator = new Curator();
-                    curator.addTask("vscan").curate(item);
+                    curator.addTask("vscan").curate(context, item.getHandle());
```

Comment:
Indeed, I'm mystified how this ever worked correctly. I ran into trouble setting up the vscan step on DSpace 5.x and rather hopefully deferred further attempts until after an upgrade to 6.x. That hope seems to have been misplaced. I agree, we probably need to augment the Curator.curate... though it's used in a few places, so that counts as a refactor.